### DISCUSSION TITLE
Take some actions after user registered with Provider(google)

### DISCUSSION THREAD

**Discussion by fellz** [strapi/strapi#7486] [Legacy Topics]
Hi
I want to perform some actions ( create resource) after i got authentication from provider ( google in my case). How to do that ? Is there any way to customize Auth controller and register action ? I found that this controller in `user-permissions` plugin in Auth.js controller -> register action.

**Comment by lauriejim** [comment]
Hello! you can plug your code in the lifecycle function afterCreate of the User model.
You will have an attribute named `provider` with the value `google` when a user is created via the Google provider.
